loop/routes/rooms.js

```diff
@@ -339,11 +341,13 @@ module.exports = function (apiRouter, conf, logError, storage, auth,
                           "clientMaxSize should be a number.");
                 return;
               }
+              var channel = req.roomStorageData.channel;
               var ttl = roomsConf.participantTTL;
               var role = req.user === roomOwnerHmac ? 'moderator' : 'publisher';
               var sessionToken = tokBox.getSessionToken(
                 req.roomStorageData.sessionId,
-                role
+                role,
+                {channel: channel}
```

Comment:
We probably should pass all the options in the options dict (and not have two different ways of passing arguments).